lib/matplotlib/backend_bases.py

```diff
@@ -508,30 +508,31 @@ def get_image_magnification(self):
         """
         return 1.0
 
-    def draw_image(self, gc, x, y, im, trans=None):
+    def draw_image(self, gc, x, y, im, transform=None):
```

Comment:
The point is that if anybody passes `trans=` as a kwarg, it is already broken:
```
lib/matplotlib/backends/backend_pgf.py:    def draw_image(self, gc, x, y, im, transform=None):
lib/matplotlib/backends/backend_template.py:    def draw_image(self, gc, x, y, im):
lib/matplotlib/backends/backend_ps.py:    def draw_image(self, gc, x, y, im, transform=None):
lib/matplotlib/backends/backend_svg.py:    def draw_image(self, gc, x, y, im, transform=None):
lib/matplotlib/backends/backend_gdk.py:    def draw_image(self, gc, x, y, im):
lib/matplotlib/backends/backend_pdf.py:    def draw_image(self, gc, x, y, im, transform=None):
lib/matplotlib/backends/backend_wx.py:    def draw_image(self, gc, x, y, im):
lib/matplotlib/backends/backend_cairo.py:    def draw_image(self, gc, x, y, im):
lib/matplotlib/backend_bases.py:    def draw_image(self, gc, x, y, im, trans=None):
```
The base class is *the only one* calling it `trans`, which leaves a trap for anybody using `**dd` to pass in the keyword arguments.